src/qililab/circuit_transpiler/circuit_transpiler.py

```diff
@@ -12,39 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""This file contains the functions used to decompose a circuit into native gates and to compute virtual-Z gates."""
-from qibo import gates
-from qibo.models import Circuit
+"""Circuit Transpiler class"""
 
-from qililab.settings.runcard import Runcard
-from qililab.circuit_transpiler.gate_decompositions import translate_gates
-
-from .native_gates import Drag, Wait
-from typing import TYPE_CHECKING
-
-if TYPE_CHECKING:
-    from qililab.platform import Platform, Bus
 
 import contextlib
 from dataclasses import asdict
+from typing import TYPE_CHECKING
 
 import numpy as np
+from qibo import gates
 from qibo.gates import Gate, M
+from qibo.models import Circuit
 
 from qililab.chip.nodes import Coupler, Qubit
+from qililab.circuit_transpiler.gate_decompositions import translate_gates
 from qililab.constants import RUNCARD
-from qililab.instruments import AWG
 from qililab.settings.gate_event_settings import GateEventSettings
 from qililab.typings.enums import Line
 from qililab.utils import Factory
 
 from ..pulse.pulse import Pulse
 from ..pulse.pulse_event import PulseEvent
 from ..pulse.pulse_schedule import PulseSchedule
+from .native_gates import Drag, Wait
+
+if TYPE_CHECKING:
```

Comment:
I don't like this at all. I prefer not having type checks for this specific case than creating this variable